dist/dorsal.js

```diff
@@ -207,7 +207,15 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
+    var pluginKeys = Object.keys(this.plugins);
+
+    if (!pluginKeys.length) {
+        if (console && console.warn) {
```

Comment:
I was just done implementing log.warn for this suggestion. Then I noticed there's no way to put Dorsal into debug mode without modifying the constant, effectively making the whole custom logging thing useless until enabling debug mode by Dorsal consumers is implemented.

The message in question is pretty critical to usage of Dorsal, and is useful in a development environment (at least ours).

I'll commit my updates to DorsalLog, enabling `log.info` and `log.warn`, but will keep this explicit `console.warn` until we have ratified enabling debug mode for library consumers. I don't have the cycles at right now to go through that too.